src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpThurloeDAO.scala

```diff
@@ -97,17 +79,13 @@ class HttpThurloeDAO ( implicit val system: ActorSystem, implicit val executionC
 object HttpThurloeDAO {
   // Not private to allow this to be called by tests.
   def calculateExpireTime(profile:Profile): Long = {
```

Comment:
I actually think this function should be inlined and the HttpThurloeDAO object deleted.